package.json

```diff
@@ -91,7 +91,7 @@
     "@chbphone55/classnames": "^2.0.0",
     "@lingui/core": "^4.3.0",
     "@warp-ds/core": "^1.0.0",
-    "@warp-ds/css": "^1.1.2",
+    "@warp-ds/css": "^1.1.3-next.1",
```

Comment:
When we get this out we should be able to bump the version to 1.2.0: https://github.com/warp-ds/css/pull/75